libcontainer/selinux/selinux.go

```diff
@@ -58,16 +57,33 @@ func getSelinuxMountPoint() string {
 	}
 	selinuxfs = ""
 
-	mounts, err := mount.GetMounts()
+	f, err := os.Open("/proc/self/mountinfo")
 	if err != nil {
 		return selinuxfs
 	}
-	for _, mount := range mounts {
-		if mount.Fstype == "selinuxfs" {
-			selinuxfs = mount.Mountpoint
-			break
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		txt := scanner.Text()
+		sepIdx := strings.Index(txt, " - ")
+		if sepIdx == -1 {
+			continue
+		}
+		if len(txt) < sepIdx+12 {
+			continue
+		}
+		if txt[sepIdx+3:sepIdx+12] != "selinuxfs" {
```

Comment:
@cyphar @LK4D4 Thank you very much for your helpful suggestions. I think eliminating the magic numbers makes sense in this context to run a large number of containers, since the primary overhead of `mount.GetMounts()` comes from parsing all lines in `mountinfo` and creating corresponding objects. Fixed to use `strings.Contains()`.